src/test/java/org/terasology/structureTemplates/components/BlockRegionBoundingBoxTest.java

```diff
@@ -46,9 +47,9 @@ public void testBoundingBox() {
         blockRegionA.regionsToFill.add(regionA);
         blockRegionA.regionsToFill.add(regionB);
         blockRegionA.regionsToFill.add(regionC);
-        Region3i bb = blockRegionA.getBoundingBox();
-        assertEquals(bb.max(), new Vector3i(6, 1, 5));
-        assertEquals(bb.min(), new Vector3i(2, 0, -1));
+        Region3i bb = BlockRegionUtilities.getBoundingBox(blockRegionA);
+        Assert.assertEquals(bb.max(), new Vector3i(6, 1, 5));
```

Comment:
oh, that accounts probably accounts to me advising to avoid static imports... I guess static imports are fine for `Assert` in unit tests, but `BlockRegionUtils` should not be statically imported imho.